http/vibe/http/common.d

```diff
@@ -618,11 +620,39 @@ FreeListRef!ChunkedOutputStream createChunkedOutputStreamFL(OS)(OS destination_s
 	return FreeListRef!ChunkedOutputStream(interfaceProxy!OutputStream(destination_stream), allocator, true);
 }
 
+/// Parses the cookie from a header field, returning the name of the cookie.
+Cookie parseHTTPCookie(string headerString) 
```

Comment:
There is an extra space at the end of the line here, which makes the travis build fail.